### PR TITLE
data: Check test names are unique

### PR DESCRIPTION
**Pull request by spbnick**

### CODE REVIEW

**Comment by spbnick** [comment]
@cki-bot, please test!

**Comment by spbnick** [comment]
@wizardofoz123, do you think you could review this today?

File: kpet/data.py [hunk 1]
```diff
@@ -553,6 +554,20 @@ def get_sets_from_regex_list(regex_list):
...
                                   format(suite.name, case.name))
 
+    def validate_test_names(self):
+        """
+        Check test names are unique.
+        """
+        name_list = []
+        for suite in self.suites:
+            for case in suite.cases:
+                name_list.append(tuple(filter(lambda x: x is not None,
```
Comment (wizardofoz123) [inline kpet/data.py:564]: This isn't something I'd do, the semantics look a bit complicated. But I'm fine with leaving as-is, if the tests pass.

**Comment by wizardofoz123** [comment]
Code makes sense. Looks like the CI is still broken?

**Review by wizardofoz123 — APPROVED** [review]

File: kpet/data.py (hunk 1 above)
Comment (spbnick) [reply kpet/data.py:564]: Yeah, this is non-trivial and kind of an overkill for the two-value case. However, we'll have to support arbitrary number of values soonish and this approach will be more appropriate, so I'd rather keep it as is.
Speaking of tests, I don't actually have any, and this is an important feature, so I'd go and write some.
Thanks :)!

**Comment by spbnick** [comment]
Yeah, CI works with #136 (which uses a different branch name), so I'll close this one and continue there. Thanks for the review!

**Pull request closed without merging**